Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM php:7.1-cli
+RUN curl https://getcomposer.org/download/1.5.2/composer.phar -o /usr/local/bin/composer && chmod +x /usr/local/bin/composer
+COPY . /opt/proofreader
```

Comment:
I think its more efficient to do this after `composer install` (so only copy `composer.json` and `composer.lock` here)? Though as it's a small repo other thinks don't change much.